lightkurve/seismology.py

```diff
@@ -0,0 +1,206 @@
+"""Defines the asteroseismology module"""
+from __future__ import division, print_function
+
+import copy
+import os
+import logging
+import warnings
+
+import numpy as np
+from matplotlib import pyplot as plt
+import astropy
+from astropy import units as u
+from astropy import constants as const
+
+from uncertainties import ufloat
+from uncertainties import umath
+
+log = logging.getLogger(__name__)
+
+__all__ = ['estimate_radius','estimate_mass','estimate_logg']
+
+"""Global parameters for the sun"""
+numaxsol = ufloat(3090, 30) # microhertz | Huber et al. 2011
+dnusol = ufloat(135.1, 0.1) # microhertz | Huber et al. 2011
+teffsol = ufloat(5772., 0.8) # Kelvin    | Prša et al. 2016
+gsol = ((const.G * const.M_sun)/(const.R_sun)**2).to(u.cm/u.second**2) #cms^2
```

Comment:
Nitpicking: [Python PEP8 convention](https://www.python.org/dev/peps/pep-0008/#constants) is to use UPPERCASE_VARIABLE_NAMES for constant values.  While not strictly required for functionality, it is good to use uppercase here for code readability!

Reply:
Thats an easy fix :)